index.js

```diff
@@ -1,4 +1,5 @@
 const mongoose = require('mongoose');
+mongoose.set('useFindAndModify', false);
```

Comment:
You can set this value in the `options` object in the `db.config.js` file. The same place where `useNewUrlParser` is passed